### PR TITLE
upgrade ruby to 3.x and add/bump dependencies

### PR DESCRIPTION
**Pull request by rshewitt**
related to [#4514](https://github.com/GSA/data.gov/issues/4514)

### CODE REVIEW

**Comment by rshewitt** [comment]
@btylerburton

**Comment by btylerburton** [comment]
is this building successfully @rshewitt?

File: Staticfile
Comment (btylerburton) [inline Staticfile:1]: what was this file for? is it necessary to build?

Comment (btylerburton) [reply Staticfile:1]: from pages documentation: https://cloud.gov/docs/management/static/

**Review by btylerburton — APPROVED** [review]

**Pull request merged**